services/nftStorage.ts

```diff
@@ -48,7 +49,18 @@ export const getKey = async (address: string): Promise<PinningKey> => {
   return { expiry, token }
 }
 
-export const pinFileToIPFS = async (file: Blob, _: string): Promise<string> => {
```

Comment:
Why not create new function?

Reply:
why yes?

second variable is not used by the function, just giving a false signature

Reply:
Isnt this function used for minting/create collection?

Reply:
it is .
following your comment:
https://github.com/kodadot/nft-gallery/issues/7184#issuecomment-1711712918
>This call can be skipped tbh

![image](https://github.com/kodadot/nft-gallery/assets/22791238/f2d529de-ba7f-427e-b576-dda66ea48064)


I've removed the call to `fetchPinningKey`
and not pass `token` into `pinFileToIPFS` - which wasn't using it anyway